Example/Pods/Target Support Files/Pods-TABObserverSet_Example-TABObserverSet_Tests/Pods-TABObserverSet_Example-TABObserverSet_Tests-acknowledgements.markdown

```diff
@@ -0,0 +1,22 @@
+# Acknowledgements
+This application makes use of the following third party libraries:
+
+## TABObserverSet
+
+ObserverSet is distributed under a BSD license, as listed below.
+
+
+Copyright (c) 2015, Michael Ash
```

Comment:
Here's where it gets interesting! I have no idea. I was hoping someone would know. If we change  that notice, are we not breaking the terms of the original license?